src/platform/user/selectors.js

```diff
@@ -10,6 +10,7 @@ import {
 export const selectUser = state => state.user;
 export const isLoggedIn = state => selectUser(state).login.currentlyLoggedIn;
 export const selectProfile = state => selectUser(state)?.profile || {};
+export const isVAPatient = state => selectProfile(state).vaPatient === true;
```

Comment:
@department-of-veterans-affairs/frontend-review-group This addition plus the supporting unit tests are the only Platform code that needs your review.